.travis.yml

```diff
@@ -51,3 +51,21 @@ matrix:
       script:
         - bash <(curl -s https://codecov.io/bash)
         - codecov
+
+    - language: generic
+      dist: trusty
+      sudo: false
+      script: skip
+      before_deploy:
+        - openssl aes-256-cbc -K $encrypted_0139fbcb2566_key -iv $encrypted_0139fbcb2566_iv
+          -in deploy_rsa.enc -out deploy_rsa -d
+        - eval "$(ssh-agent -s)"
+        - cp deploy_rsa ~/.ssh/deploy_rsa
+        - chmod 600 ~/.ssh/deploy_rsa
+        - ssh-add ~/.ssh/deploy_rsa
+      deploy:
+        - provider: script
+          skip_cleanup: true
+          script: bash scripts/deploy_hcloud.sh
+          on:
+            all_branches: true
```

Comment:
(hopefully last comment) what happens if this is merged and nothing is set up (no deplyment ssh key, no env vars etc)? This should not disturb travis CI testing. Only in case that all necessary parts are available, the deployment should be done. Can you look into that?

Reply:
@norbusan one test is failing
ai.susi.mind.SusiTutorialTest > test FAILED
    java.lang.AssertionError at SusiTutorialTest.java:86

Reply:
Thanks, that looks fine.